logger.c

```diff
@@ -186,9 +186,9 @@ static int _logger_thread_parse_ise(logentry *e, char *scratch) {
 
     uriencode(le->key, keybuf, le->nkey, KEY_MAX_URI_ENCODED_LENGTH);
     total = snprintf(scratch, LOGGER_PARSE_SCRATCH,
-            "ts=%d.%d gid=%llu type=item_store key=%s status=%s cmd=%s ttl=%u clsid=%u\n",
+            "ts=%d.%d gid=%llu type=item_store key=%s status=%s cmd=%s ttl=%u clsid=%u connid=%d\n",
```

Comment:
s/connid/cfd/ or cid? it's the actual file descriptor; either way connid is a bit long considering 'gid' and 'clsid' are shortened.